client/landing/stepper/declarative-flow/internals/steps-repository/sensei-launch/launch-completion-tasks.ts

```diff
@@ -0,0 +1,34 @@
+import wp from 'calypso/lib/wp';
+
+const PERSISTENT_DATA_DELAY = 1200;
```

Comment:
Just to check...
I noticed the other steps have 3 - 5 seconds. From your tests and debugging, do you think `1200` is safe? The waits there have different purposes than this one?